src/v/security/audit/probe.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+
+#include "seastarx.h"
+#include "ssx/metrics.h"
+
+#include <seastar/core/lowres_clock.hh>
+#include <seastar/core/metrics_registration.hh>
+
+namespace security::audit {
+
+class audit_log_manager;
```

Comment:
Maybe we should have this in `security/audit/fwd.h`? or maybe thats overkill since theres only one class in that directory

Reply:
sounds reasonable to me